.github/workflows/ci-linux.yml

```diff
@@ -66,124 +43,21 @@ jobs:
       targets_pre: all-sage-local
       targets: build doc-html
       targets_optional: ptest
+      tox_system_factors: >-
+          ["ubuntu-jammy"]
       tox_packages_factors: >-
```

Comment:
At the end, you renamed "default" to "minimal", effectively removing "minimal" configurations. 

The original "minimal" configurations serve a purpose, as discussed in the other PR.

Reply:
The "minimal" configurations are intended to test standard sage packages.